framework/helpers/BaseHtml.php

```diff
@@ -430,11 +430,22 @@ public static function mailto($text, $email = null, $options = [])
      * the attributes of the resulting tag. The values will be HTML-encoded using [[encode()]].
      * If a value is null, the corresponding attribute will not be rendered.
      * See [[renderTagAttributes()]] for details on how attributes are being rendered.
+     * @since 2.0.12 It is possible to pass the "srcset" option as an array which keys are
+     * descriptors and values are URLs. All URLs will be processed by [[Url::to()]].
```

Comment:
Interesting, but this is not how our apidoc uses it :)
We currently use `@since` only to indicate when a class or method has been introduced.
Seems this is some kind of changelog for the method, which is a nice idea as we could probably use it to generate more detailed changelogs and provide information on which methods have changed in a release. Its not how it works currently though, so if we start doing this we need to adjust it in other places also and apply this approach to the whole framework.